packages/leaderboard/src/board.ts

```diff
@@ -0,0 +1,213 @@
+      [
+        JSON.stringify(
+          lst.map((a) => {
+            const res = { address: addressToBinary(address), claimed_address: addressToBinary(a) }
+            console.log(res)
+            return res
+          })
+        ),
+      ]
+    )
+    console.log(res.rows)
+    await client.end()
+  } catch (err) {
+    console.error('Cannot process claims', err)
+  }
+}
+
+readSheet()
+
+function readSheet() {
+  // Load client secrets from a local file.
+  fs.readFile('credentials.json', (err, content) => {
+    if (err) return console.log('Error loading client secret file:', err)
+    // Authorize a client with credentials, then call the Google Sheets API.
+    authorize(JSON.parse(content.toString()), getInfo)
+  })
+}
+
+/**
+ * Create an OAuth2 client with the given credentials, and then execute the
+ * given callback function.
+ * @param {Object} credentials The authorization client credentials.
+ * @param {function} callback The callback to call with the authorized client.
+ */
+function authorize(credentials: any, callback: any) {
+  const { client_secret, client_id, redirect_uris } = credentials.installed
+  const oAuth2Client = new google.auth.OAuth2(client_id, client_secret, redirect_uris[0])
+
+  // Check if we have previously stored a token.
+  fs.readFile(TOKEN_PATH, (err, token) => {
+    if (err) return getNewToken(oAuth2Client, callback)
+    oAuth2Client.setCredentials(JSON.parse(token.toString()))
+    callback(oAuth2Client)
+  })
+}
+
+/**
+ * Get and store new token after prompting for user authorization, and then
+ * execute the given callback with the authorized OAuth2 client.
+ * @param {google.auth.OAuth2} oAuth2Client The OAuth2 client to get token for.
+ * @param {getEventsCallback} callback The callback for the authorized client.
+ */
+function getNewToken(oAuth2Client: any, callback: any) {
+  const authUrl = oAuth2Client.generateAuthUrl({
+    access_type: 'offline',
+    scope: SCOPES,
+  })
+  console.log('Authorize this app by visiting this url:', authUrl)
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  })
+  rl.question('Enter the code from that page here: ', (code) => {
+    rl.close()
+    oAuth2Client.getToken(code, (err: any, token: any) => {
+      if (err) return console.error('Error while trying to retrieve access token', err)
+      oAuth2Client.setCredentials(token)
+      // Store the token to disk for later program executions
+      fs.writeFile(TOKEN_PATH, JSON.stringify(token), (err) => {
+        if (err) return console.error(err)
+        console.log('Token stored to', TOKEN_PATH)
+      })
+      callback(oAuth2Client)
+    })
+  })
+}
+
+function getInfo(auth: any) {
+  const sheets = google.sheets({ version: 'v4', auth })
+  sheets.spreadsheets.values.get(
+    {
+      spreadsheetId: '1d3pZNof8p3z8M9O3MH5FZG5dA3e-L52XiJ4qA5o7X0Y',
+      range: 'Sheet1!A2:B',
```

Comment:
we have to make sure that's the spreadsheet convention (link to template in README)